system/Test/bootstrap.php

```diff
@@ -78,7 +79,7 @@
 
 // Use Config\Services as CodeIgniter\Services
 if (! class_exists('CodeIgniter\Services', false)) {
-    class_alias('Config\Services', 'CodeIgniter\Services');
+    class_alias(ServicesConfig::class, 'CodeIgniter\Services');
```

Comment:
ref https://3v4l.org/sRoiH aliased class with `::class` still point to original class of it.

Reply:
@MGatner @kenjis @paulbalandan  on `system/Test/bootstrap.php`, the next line: 

```php
// Initialize and register the loader with the SPL autoloader stack.
Services::autoloader()->initialize(new Autoload(), new Modules())->register();
```

is pointed to `CodeIgniter\Services`, while in `system/bootstrap.php`, that's pointed to `Config\Services`, is that on purpose?

Reply:
I noticed this too before. I think it was just to show that the class aliasing works?

Reply:
For CI4 testing, `CodeIgniter\Services` is okay. But if a user overrides `autoloader()` in `Config\Services`, it is not okay.

Reply:
@kenjis do you mean the current implementation is correct?

Reply:
No, we should use `Config\Services` in both.

Reply:
ok, updated to use `Config\Services` https://github.com/codeigniter4/CodeIgniter4/pull/6052/commits/040e94e1db4a3664274cd08522b82e0c907c6c39

Reply:
https://github.com/codeigniter4/CodeIgniter4/blob/develop/tests/_support/Services.php
I don't know where this class is loaded.